core/src/model/Ad4mModel.ts

```
@@ -716,22 +734,22 @@ export class Ad4mModel {
           switch (value.action) {
             case "setter":
-              await this.setCollectionSetter(key, value.value);
+              await this.setCollectionSetter(key, value.value, batchId);
               break;
             case "adder":
-              await this.setCollectionAdder(key, value.value);
+              await this.setCollectionAdder(key, value.value, batchId);
               break;
             case "remover":
-              await this.setCollectionRemover(key, value.value);
+              await this.setCollectionRemover(key, value.value, batchId);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)